mockup_package/mockup/image_generator.py

```diff
@@ -113,7 +116,14 @@ def create_fit_coord_image(self, spec):
         )
         distort_img = cv2.cvtColor(distort_img, cv2.COLOR_BGR2RGB)
         distort_img = Image.fromarray(distort_img)
-        distort_img.save(self.tmp_result_image_path)
+
+        distort_img_with_padding = Image.new(
+            "RGBA",
+            (distort_img.size[0] + 2 * PADDING, distort_img.size[1] + 2 * PADDING),
+            (0, 0, 0, 255),
+        )
+        distort_img_with_padding.paste(distort_img, (PADDING, PADDING))
+        distort_img_with_padding.save(self.tmp_result_image_path)
```

Comment:
As per offline discussion, seems this logic is to 

> apply extra padding ONLY AFTER distortion (rotation)

Can you help
1. add comments to explain this?
2. test un-rotated images are unaffected by this change?

Thanks!

Reply:
[screenshots.zip](https://github.com/user-attachments/files/16595097/screenshots.zip)

fit-size screenshots 🙏

Reply:
@pkong-ds testing using the fit-size image, will still see the strange triangle because there is a black area between the phone frame and the display area
![samsung-s24-ultra-right](https://github.com/user-attachments/assets/75936a3a-89b9-4885-874f-dfac4a0d1d0d)

here is the B/A result:
type | Before | After
------|--------|------
landscape | ![samsung-s24-ultra-landscape](https://github.com/user-attachments/assets/b3ec7528-20a7-4508-b8c2-f7b6308e59c8) | ![samsung-s24-ultra-landscape](https://github.com/user-attachments/assets/bb508211-edd4-4dc3-b5d1-ed228150c12b)
portrait | ![samsung-s24-ultra-portrait](https://github.com/user-attachments/assets/a9f56fec-6524-4983-9d21-2c598e1094a5) | ![samsung-s24-ultra-portrait](https://github.com/user-attachments/assets/96c6046e-be8e-4b36-8a2f-b374ffbe7d79)
left | ![samsung-s24-ultra-left](https://github.com/user-attachments/assets/5e90d39b-40ff-44b8-96c1-e8479a17a9f5)  | ![samsung-s24-ultra-left](https://github.com/user-attachments/assets/e33e5a43-69d9-494c-acae-dc371abe7406)
right | ![samsung-s24-ultra-right](https://github.com/user-attachments/assets/8d258f18-bfd9-47a9-8258-3285f1a01b8a) | ![samsung-s24-ultra-right](https://github.com/user-attachments/assets/15768da2-e0af-442e-9c62-f99510c485e4)

For the portrait one, the gray area on the top disappear after the update, but I think that should be expected, I guess the `coords` in `device_info.json` is incorrect so the image is pasted slightly to the bottom than expected , but it is not related to the current issue. (not sure if we need to spend some time checking that all model templates and configurations are correct)

Reply:
> For the portrait one, the gray area on the top disappear after the update

What gray area disappear? They look same to me :sadfrog:

<img width="792" alt="image" src="https://github.com/user-attachments/assets/5dba85fa-12fc-4a94-97e9-2d9df574b8b2">

Reply:
@pkong-ds opps, should be `landscape` one, sorry

Reply:
Ahh, I see it now. 🙏 

https://github.com/user-attachments/assets/002758fb-614b-4b7e-9854-2123de6247dd



> but I think that should be expected

Why do you think this is expected in `AFTER`? What did you padding do to make this? Can huddle if too complicated

> not sure if we need to spend some time checking that all model templates and configurations are correct

I spent some time checked already, seems still have problem. Can help open issue for this device first? 🙇